### ISSUE TITLE
Multi-species: Adding GMT file to Yeast dataset drastically reduces number of genes

### ISSUE THREAD

**Issue by AxelBigOmics** [bigomics/omicsplayground#783]
I just tried running the platform with a yeast sample and a GMT file and something odd happened. Instead of the around 5000 genes I was expecting to see (as when I did the same run without GMT), the platform now only displays 712:

![image](https://github.com/bigomics/omicsplayground/assets/85561624/20e6fdc9-4346-4ea8-ac55-5179062321c3)

I also had a break-down message if I tried running the dataset with a GMT file but without the "transform features to gene symbols" option.

**Comment by phisanti** [comment]
Hi @AxelBigOmics, it looks like the table shows the organism selected is "human". Have you checked that?

**Comment by AxelBigOmics** [comment]
True...I just tried it again and it worked. I think the error was due to me choosing the wrong species. Please close this thread.
